flink-end-to-end-tests/flink-tpch-test/src/main/java/org/apache/flink/table/tpch/TpchResultComparator.java

```diff
@@ -81,7 +93,10 @@ public static void main(String[] args) throws IOException {
                                 failed = (e * 0.99 > t || e * 1.01 < t);
                             }
                         } catch (NumberFormatException nfe2) {
-                            failed = !expected[i].trim().equals(actual[i].trim());
+                            failed =
+                                    !expected[i]
+                                            .trim()
+                                            .equals(actual[i].replaceAll("\"", "").trim());
```

Comment:
Why there are quotes on numeric?

Reply:
In the new data type system, we will use qoutoes for String type, e.g. "string value", which is different from the behaviour before. Therefore, we remove the quotes when we can't convert the value to `long` and `double`.